content/docs/releases/upgrading.mdx

```diff
@@ -13,6 +13,40 @@ sidebar_position: 10
 
 # Upgrade guide
 
+## Since 0.20.0
+
+### Upgrading
+
+There are several data model changes in this release that are not backward compatible. Please make sure you back up your Postgres database before performing an upgrade.
+
+### Breaking
+
+#### Devices need be re-enrolled
+
+[Devices (beta)](/docs/capabilities/device-identity) data model had an internal change that is not forward compatible. Your enrolled devices need be re-registered, and their IDs would change. Your existing policies may need be updated.
+
+#### (TBD)
+
+confirm sessions can rolled from 0.20 to 0.21 after auth changes
+
+#### Forward Auth (deprecated, removed in this release)
```

Comment:
\cc @ZPain8464  to update documentation to reflect this is now deprecated. Remove references.